src/Development/Rattle/Profile.hs

```diff
@@ -0,0 +1,252 @@
+{-# LANGUAGE ScopedTypeVariables, RecordWildCards, TupleSections, ViewPatterns #-}
+
+module Development.Rattle.Profile(
+  constructGraph, Graph(..), dotStringOfGraph,
+  graphData, writeProfile
+  ) where
+
+import Development.Rattle.Server
+import Development.Rattle.Types
+import Development.Rattle.Hash
+import Development.Rattle.Shared
+import Control.Monad
+import System.IO.Unsafe(unsafeInterleaveIO)
+import Data.Maybe as Maybe
+import Data.List as List
+import qualified Data.ByteString.Lazy.Char8 as LBS
+import General.Template
+import General.Paths
+import Data.HashMap.Strict as Map
```

Comment:
Almost always import HashMap qualified

Reply:
You mean change Map to HashMap?

Reply:
`import qualified Data.HashMap.Strict as Map` is the standard idiom.